client/containers/Pub/SpubHeader/SpubHeaderToolBar/spubHeaderToolbar.scss

```diff
@@ -0,0 +1,40 @@
+@import 'styles/variables.scss';
+
+.spubheader-toolbar {
+	display: flex;
+	justify-content: center;
+	align-items: baseline;
+	column-gap: 340px;
+	.tabs.spub-header-component .bp3-tab-list {
+		margin: 0px;
+	}
+	.spub-header-component .bp3-tab-list .bp3-tab.inactive {
+		opacity: 0.45;
+		color: var(--community-accent-dark);
+	}
+}
+
```

Comment:
As a matter of form, everything inside of a component's SCSS definition _must_ be nested inside the outermost class wrapper. there are very few exceptions to this rule (like when React renders something outside of the component HTML using a Portal). As written this would modify any element on the page that happens to have the class `.status`.